binary_gentoo/internal/cli/tree_diff.py

```diff
@@ -36,13 +108,39 @@ def report_new_and_changed_ebuilds(config):
         print(cpv)
 
 
+def enrich_config(config):
+    if config.keywords is None:
+        config.keywords = announce_and_check_output(['portageq', 'envvar',
+                                                     'ACCEPT_KEYWORDS']).rstrip()
+    if not config.keywords:
+        raise ValueError("At least one keyword must be specified")
+
+    # add stable keywords for testing keywords
+    config.keywords = {kw for kw in config.keywords.split(" ") if kw}
+    config.keywords |= {k[1:] for k in config.keywords if k.startswith('~')}
+
+    return config
+
+
 def parse_command_line(argv):
     parser = ArgumentParser(
         prog='gentoo-tree-diff',
         description='Lists packages/versions/revisions that one portdir has over another')
 
     add_version_argument_to(parser)
 
+    parser.add_argument('--keywords',
+                        help='include only packages/versions/revisions that have these keywords; '
+                        'in case of multiple keywords a space-separated list can be provided '
+                        '(default: auto-detect using portageq)')
+
+    parser.add_argument('--report-changes',
```

Comment:
Would you mind if we rename this to e.g. `--all-changes`? Because we report certain changes even without the flag given.

Reply:
I don't mind. What do you think of the help text? I found it quite difficult to explain clearly what it does...

Reply:
I agree it's a bit tricky, and if `--all-changes` will still be filtered further by keywords, always, maybe `--all-changes` will not be a good name then :thinking: 

How about…

> `--body-changes`
> include ebuilds that had non-keyword content changes
> (default: only include previously existing ebuilds when relevant keywords have been added)

…or…

> `--pessimistic`
> be more robust towards missing revbumps 
> by including ebuilds that had non-keyword content changes
> (default: only include previously existing ebuilds when relevant keywords have been added)

?

Reply:
Let's go with the `--pessimistic` then.